src/reducers.js

```diff
@@ -49,10 +49,19 @@ function organisations(
     timesFetched: 0,
     available: [],
     availableForRasterSharedWith: [],
-    selected:
-      JSON.parse(
-        localStorage.getItem("lizard-management-current-organisation")
-      ) || null
+    selected: (function () {
+      const localStorageResult = localStorage.getItem("lizard-management-current-organisation");
+      if (!localStorageResult) {
+        return null;
+      }
+      let result;
+      try {
+        result = JSON.parse(localStorageResult);
+      } catch(e) {
+          return null;
+      }
+      return result;
+    }()),
```

Comment:
This should be an util function like `getLocalStorageJSON("lizard-management-current-organisation", null)`.

Reply:
created a ticket for this

Reply:
done